src/main/kotlin/no/item/gradle/beans/InputType.kt

```diff
@@ -0,0 +1,20 @@
+package no.item.gradle.beans
+
+enum class InputType {
```

Comment:
The convention for `enum` is that constants are upper case. Ref. https://kotlinlang.org/docs/reference/enum-classes.html

Reply:
Alright, lets keep it to convention.